app/views/shopify_app/partials/_button_styles.html.erb

```diff
@@ -29,30 +30,44 @@
     user-select:none;
     text-decoration:none;
     transition-property:background, border, box-shadow;
-    transition-duration:200ms;
+    transition-duration:.1s;
```

Comment:
might be easier to read as `100ms` instead; since someone could parse this as 1 second